sys/include/net/gcoap.h

```diff
@@ -739,6 +748,9 @@ int gcoap_req_init(coap_pkt_t *pdu, uint8_t *buf, size_t len,
  * Assumes the PDU has been initialized with a gcoap_xxx_init() function, like
  * gcoap_req_init().
  *
+ * @deprecated  Will be removed after the 2020.04 release. Use
```

Comment:
Don't we usually remove after two cycles? We are in the middle of the 2020.01 release cycle, so the proper removal release would be 2020.07.

Reply:
Yes, that's what I was trying to say. When I read, "after the 2020.04 release", that implies sometime before 2020.07 is released. I adopted this expression from other deprecations I had seen. Perhaps I misunderstood.

IMO it would be clearer to say "Will be removed for the 2020.07 release." What do you think?

Reply:
> IMO it would be clearer to say "Will be removed for the 2020.07 release." What do you think?

Yes, that is typically what we mean with "after two cycles": Starting from the current one under development + 2 releases. The "after" refers to the fact that it will be removed during the release development and thus not be available *after* the release is out.

Reply:
Just to be clear, I hear you say that you prefer:

> Will be removed after the 2020.07 release.

rather than

> Will be removed for the 2020.07 release.

To me "removed" is an active _verb_ that means "the act of taking it out". So, it sounds like you are saying the act of taking out the function will happen after the release.

So, to keep the word "after" it would be clearer to use what you stated in your comment:

> Will be unavailable after the 2020.07 release.

Reply:
You are the native speaker ^^" and what you say makes sense to me. Then let's go for "unavailable" or "not available" (the latter I prefer somehow, can't really explain why, though semantically they have the same meaning).